qa/workunits/suites/fsx.sh

```diff
@@ -4,6 +4,7 @@ set -e
 
 git clone https://git.ceph.com/xfstests-dev.git
 cd xfstests-dev
+git checkout 0e5c12dfd008efc2848c98108c9237487e91ef35
```

Comment:
nit: comment about why _this_ sha is being used.

Reply:
Done. PTAL.

Reply:
Why not take the tip of `main`? Are we worried that could be broken? It must be used elsewhere, so someone will fix it if it breaks. Otherwise, we're going to miss new tests or improvements pushed to the tests

Reply:
There is no need to consider the `new tests`, for this we only care about the `fsx` bin in the `xfstests-dev`, please see:

```
cp xfstests-dev/ltp/fsx .

OPTIONS="-z"  # don't use zero range calls; not supported by cephfs

./fsx $OPTIONS  1MB -N 50000 -p 10000 -l 1048576
./fsx $OPTIONS  10MB -N 50000 -p 10000 -l 10485760
./fsx $OPTIONS 100MB -N 50000 -p 10000 -l 104857600
```
The `xfstests-dev` repo in `https://git.ceph.com/` will be pulled from upstream periodically everyday as I remember. And always using the latest code is too aggressive and may cause new compilation or unknown errors.

Reply:
ack